src/Platforms/AbstractPlatform.php

```diff
@@ -1162,6 +1167,10 @@ public function getCreateSchemaSQL(string $schemaName): string
             throw NotSupported::new(__METHOD__);
         }
 
+        if (preg_match('/^\d/', $schemaName)) {
+            return 'CREATE SCHEMA ' . $this->quoteIdentifier($schemaName);
+        }
```

Comment:
at first, I was thinking of create an asset `Doctrine\DBAL\Schema\Namespace` but namespace are handled in a different way than other assets, and it seemed easier to do it like this, but I still can create the new class